### ISSUE TITLE
Error thrown using clamdscan with scan_recursively false

### ISSUE THREAD

**Issue by taietel** [kylefarris/clamscan#11]
I'm using the default config, the only changes are to the clamdscan config_file  and scan_recursively which is set to false so I can get the lists with good / bad files. 
This is the error that I get: 
```
/var/www/aws-cleaner/node_modules/clamscan/index.js:535
                        if (info.isFile()) good_files.push(file);
                                ^

TypeError: Cannot read property 'isFile' of undefined
    at /var/www/aws-cleaner/node_modules/clamscan/index.js:535:33
    at FSReqWrap.oncomplete (fs.js:82:15)
```
It works fine using the other 'option' to get the 2 lists with good/bad files, using a file_cb function and scan_recursively set to true.

**Comment by kylefarris** [comment]
Thanks for the bug report @taietel. I'll take a look at it and see what's going on.

**Comment by kylefarris** [comment]
Alright, the bug's been patched by the following commit: 441c5a65ec7dea433bb4d2f2535dfee2556aa08d

Thanks again for the bug report and sorry for the inconvenience!
